package com.upc.excepciones2;

public class Negocio {
    public int dividir(int m, int n) throws Exception{
        return Utilitario.dividir(m,n);
    }
}
